.github/workflows/ci_docker_build_push.yml

```
@@ -83,3 +85,61 @@
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Get options
+        id: options
+        env:
+          TAG: ${{ github.event.inputs.new_tag_short }}
+          NAME: ${{ github.event.inputs.name }}
+          SHA: ${{ github.event.inputs.sha_short }}
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)